lib/utils.js

```diff
@@ -75,7 +75,10 @@ module.exports.formatNotes = ({ parsed, version, previous }) => {
 }
 
 module.exports.writeNotes = ({ notes, version, previous, file }) => {
-  const f = file || `release-notes-${previous}-${version}.md`
+  const invalidFNChars = /[\\/:"*?<>|#%$&{}!+`='" ]+/g
+  const p = previous.trim().toLowerCase().replace(invalidFNChars, '-')
+  const v = version.trim().toLowerCase().replace(invalidFNChars, '-')
+  const f = file || `release-notes-${p}-${v}.md`
```

Comment:
why not just apply this whole thing together

```js
const invalidFNChars = /[\\/:"*?<>|#%$&{}!+`='" ]+/g 
const f = (file || `release-notes-${p}-${v}.md`.toLowerCase()).replace(invalidFNChars, '-')
```

Reply:
oh ya that's true lol, just wanna trim `p` and `v` as well

Reply:
updated